ChangeLog.md

```diff
@@ -20,6 +20,7 @@ See docs/process.md for more on how version tagging works.
 
 3.1.32 (in development)
 -----------------------
+- Add support for Audio Worklets. (#16449)
```

Comment:
How about "Add initial support for..".. also should we warn that this is still experimental?   (I didn't review the change much so I'm not sure how complete it is).

Reply:
(By warning i mean at compile time with `diagnostics.warning` like we do for wasm64

Reply:
I think it is complete. It was on a branch for a long time, and tested by multiple people there, and only landed after it looked good to them all. So it's a new feature but I wouldn't call it experimental or wip.